apps/modernization-ui/src/apps/search/laboratory-report/result/table/LaboratoryReportSearchResultsTable.tsx

```diff
@@ -15,15 +15,15 @@ import {
 
 const LEGAL_NAME = { id: 'lastNm', name: 'Legal name' };
 const DATE_OF_BIRTH = { id: 'birthTime', name: 'Date of birth' };
-const SEX = { id: 'sex', name: 'Sex' };
+const SEX = { id: 'sex', name: 'Current sex' };
 const PATIENT_ID = { id: 'patientid', name: 'Patient ID' };
 
 const DOCUMENT_TYPE = { id: 'documentType', name: 'Document type' };
 const DATE_RECEIVED = { id: 'dateReceived', name: 'Date received' };
 const DESCRIPTION = { id: 'description', name: 'Description' };
 const REPORTING_FACILITY = { id: 'reportingFacility', name: 'Reporting facility' };
 const ORDERING_PROVIDER = { id: 'orderingProvider', name: 'Ordering provider' };
-const JURSIDICTION = { id: 'jurisdiction', name: 'Jurisdiction' };
+const JURISDICTION = { id: 'jurisdiction', name: 'Jurisdiction' };
```

Comment:
praise:  Thanks for fixing all the spelling mistakes!

Reply:
I installed cSpell VS extension, in case you're interested